parsers/to-css-textStyle/README.md

```diff
@@ -0,0 +1,86 @@
+    exclude?: Array<'color' | 'font-family' | 'font-size' | 'line-height' | 'letter-spacing' | 'text-align' | 'vertical-align' | 'text-transform' | 'font-variant' | 'text-decoration' | 'text-indent' | 'font' | 'fontSize' | 'lineHeight' | 'letterSpacing' | 'textAlign' | 'textTransform' | 'fontVariant' | 'textDecoration' | 'textIndent'>,
+    prefix?: string;
+    suffix?: string;
+    colorFormat?: 'rgb' | 'prgb' | 'hex' | 'hex6' | 'hex3' | 'hex4' | 'hex8' | 'name' | 'hsl' | 'hsv';
+    cssClassFormat?: 'camelCase' | 'kebabCase' | 'snakeCase';
+    fontFamilyFormat?: 'camelCase' | 'kebabCase' | 'snakeCase';
+    genericFamily?: 'serif' | 'sans-serif' | 'cursive' | 'fantasy' | 'monospace';
+    prettierConfig?: Partial<{
+      endOfLine: 'auto' | 'lf' | 'crlf' | 'cr';
+      tabWidth: number;
+      useTabs: boolean;
+    }>;
+  }
+}
+```
+
+### Options
+| parameter | Require    | type      | default    | description                                       |
+| --------- | ---------- | --------- | ---------- | ------------------------------------------------- |
+| `include`    | optional   | `Array<string>`   |  | List of properties to include in css classes |
+| `exclude`    | optional   | `Array<string>`   |  | List of properties to exclude in css classes |
+| `prefix`    | optional   | `string`   |  | A string will be append before the css class name |
+| `suffix`    | optional   | `string`   |  | A string will be append after the css class name |
+| `colorFormat`    | optional   | `rgb, prgb, hex, hex6, hex3, hex4, hex8, name, hsl, hsv`   | `rgb` | A color format applied when a text style include a color |
+| `cssClassFormat`    | optional   | `camelCase, kebabCase, snakeCase`   | `kebabCase` | The lodash function used to normalize the css class name |
+| `fontFamilyFormat`    | optional   | `camelCase, kebabCase, snakeCase`   |  | The lodash function used to normalize the font family value |
+| `genericFamily`    | optional   | `string` |  | The generic font family will be applied after the main font family |
+| `prettierConfig.endOfLine`    | optional   | `auto, lf, crlf, cr`   | `auto` | [Prettier documentation](https://prettier.io/docs/en/options.html#end-of-line) |
+| `prettierConfig.tabWidth`    | optional   | `number`   | `2` | [Prettier documentation](https://prettier.io/docs/en/options.html#tab-width) |
+| `prettierConfig.useTabs`    | optional   | `boolean`   | `false` | [Prettier documentation](https://prettier.io/docs/en/options.html#tabs) |
+
+## Usage example
+
+```json
+{
+    "name": "to-css-text-style",
+    "options": {
+      "exclude": ["color", "text-indent"],
+      "prefix": "utils-",
+      "suffix": "-text-style",
+      "genericFamily": "serif"
+    }
+}
+```
+### Result example
+
+```css
+.utils-main-text-style {
```

Comment:
I'd rather display a more "probable" text style like:
```css
.your-text-style {
    font-size: 12px;
    font-weight: 300;    
    letter-spacing: 1px;
    line-height: 1.7;
    text-transform: uppercase;
}
```

Reply:
Can you describe the problem exactly please.

Reply:
The text style you're displaying here is not very user friendly:
- The class `.utils-main-text-style` looks strange because of the word "utils". Hence my proposal `.your-text-style`
- I understand you want to display every CSS property available in the parser but I don't think every property should be used in the example.

My proposal is here to display an example that is more probable.

Reply:
If I used your example. Several options are not present like `prefix` and `suffix`. The goal of the example is to present some options. To build the example I use a real seed. It's more convenient and closed to the reality of the parser execution.